src/main/webapp/editFilesFragment.xhtml

```diff
@@ -77,113 +78,12 @@
                                 </p>
                     
                     <script>
-//<![CDATA[
```

Comment:
Jim, any particular reason to remove this `CDATA` section from the script?

Reply:
I think it was added when there were parse errors due to the content of the javascript. With most of that now in a separate file, I didn't think it was needed.